### PR TITLE
Added Optional Runtime Ordering for User Supplied Bash Scripts

### PR DESCRIPTION
**Pull request by CloudSnorkeler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ No modules.
 | <a name="input_sumologic_uri"></a> [sumologic\_uri](#input\_sumologic\_uri) | Sumologic uri | `string` | `""` | no |
 | <a name="input_volume_size"></a> [volume\_size](#input\_volume\_size) | Size of the sidecar disk | `number` | `15` | no |
 | <a name="input_vpc_id"></a> [vpc\_id](#input\_vpc\_id) | AWS VPC ID to deploy sidecar to | `string` | n/a | yes |
+| <a name="input_custom_user_data"></a> [custom\_user\_data](#input\_custom\_user\_data) | Acillary consumer supplied user-data script. Bash scripts must be added to a map as a value of the key `pre` and/or `post` denoting execution order with respect to sidecar installation. (Approx Input Size = 19KB) | `map` | `{"pre" = "", "post" = ""}` | no |
```
Comment (wcmjunior) [inline README.md:174]: This parameter is already documented in line `120`. Could you please move this change up? to keep it in alphabetical order? Also there is a typo in `Ancillary`.

File: variables_sidecar.tf [hunk 2]
```diff
@@ -154,7 +154,7 @@ variable "repositories_supported" {
 }
 
 variable "custom_user_data" {
-  description = "Auxiliary user-data script. Appended to existing user-data sidecar bootstrapping scripts (Approx Input Size = 19KB)"
-  type        = string
-  default     = ""
+  description = "Acillary consumer supplied user-data script. Bash scripts must be added to a map as a value of the key `pre` and/or `post` denoting execution order with respect to sidecar installation. (Approx Input Size = 19KB)"
```
Comment (wcmjunior) [inline variables_sidecar.tf:157]:
```suggestion
  description = "Ancillary consumer supplied user-data script. Bash scripts must be added to a map as a value of the key `pre` and/or `post` denoting execution order with respect to sidecar installation. (Approx Input Size = 19KB)"
```

File: README.md (hunk 1 above)
Comment (CloudSnorkeler) [reply README.md:174]: Ah yes, apologies 👍

**Comment by CloudSnorkeler** [comment]
@wcmjunior I fixed the README and removed duplicates. Please let me know if you see any other changes that need to be made.

**Comment by wcmjunior** [comment]
Thanks for your contribution @CloudSnorkeler ! :-)

**Review by wcmjunior — APPROVED** [review]

**Review by wcmjunior — APPROVED** [review]

**Pull request merged**